projects/person-search-index-from-delius/container/management/index-alias-management.sh

```diff
@@ -0,0 +1,133 @@
+#!/bin/bash
+#set -ex
```

Comment:
Any reason not to have `set -e` enabled?   Or even `set -eo pipefail`?  Just to make sure failures don't get ignored.